HEN_HOUSE/src/egs_utilities.mortran

```diff
@@ -365,7 +365,13 @@ IF( lnblnk1(input_file) > 0 ) [
     tmp_string = $cstring(ucode_dir) // $cstring(input_file) // '.egsinp';
     inquire(file=tmp_string,exist=ex);
     IF( ~ex ) [
-      $egs_fatal(*,'input file ',$cstring(tmp_string),' does not exist ');
+      ex = egs_strip_extension(input_file,'.egsinp');
```

Comment:
@rtownson can't we simply put this strip before line 365 (i.e., before adding `.egsinp` again?). The code would be simpler...